src/main/java/com/gmail/inverseconduit/BotConfig.java

```diff
@@ -25,6 +25,11 @@
     private final Path          javadocs;
 
     private final List<Integer> rooms;
+    
+    /**
+     * List of userids with elevated privileges
+     */
+    private final List<Integer> admins;
```

Comment:
Considering that the bot can be used in the whole SE network an `Integer` might not be large enough to hold the largest possible user-id... well that's probably not going to affect us in the foreseeable future anyways, soo ...